nexus/db-queries/src/db/queries/external_ip.rs

```diff
@@ -1068,9 +1056,10 @@ mod tests {
             let id = Uuid::new_v4();
             let instance_id = InstanceUuid::new_v4();
             let ip = context
-                .db_datastore
+                .db
+                .datastore()
                 .allocate_instance_snat_ip(
-                    &context.opctx,
+                    &context.db.opctx(),
```

Comment:
Is the `&` here and in the other cases below necessary? (Surprised clippy didn't mention this.)

Reply:
Done